grass-gis-addons/m.import.rvr/m.import.rvr.py

```diff
@@ -33,7 +33,7 @@
 # % required: yes
 # % multiple: yes
 # % label: Type of processing for which the data should be imported
-# % options: buildings analysis,green roofs,trees analysis
+# % options: buildings analysis,green roofs,trees analysis,train neural network
```

Comment:
```suggestion
# % options: buildings analysis,green roofs,trees analysis,neural network
```
I would take only neural network. Than the user can set the training percentage to 0 to get only the data structured for the application, once the network is ready.